open-hackathon/src/hackathon/database/models.py

```diff
@@ -192,12 +193,15 @@ class DockerHostServer(Base):
     vm_name = Column(String(100), unique=True, nullable=False)
     hackathon_id = Column(Integer)
     public_dns = Column(String(50))
+    public_ip = Column(String(50), unique=True)
     public_docker_api_port = Column(Integer)
     private_ip = Column(String(50), unique=True)
     private_docker_api_port = Column(Integer)
     container_count = Column(Integer, nullable=False)
     container_max_count = Column(Integer, nullable=False)
 
+    hackathon_id = Column(Integer, ForeignKey('hackathon.id', ondelete='CASCADE'))
```

Comment:
I added the same column already. Please make sure the latest code be merged and there won't be duplicated columns